synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -361,29 +362,32 @@ StateBone_Context::StateBone_Context(CanvasView *canvas_view) :
 
 	get_canvas_interface()->set_state("bone");
 
-	// Set up state_bone Toolbox window
-	// Title
+	// Toolbox widgets
 	title_label.set_label(_("Skeleton Tool"));
 	Pango::AttrList list;
 	Pango::AttrInt attr = Pango::Attribute::create_attr_weight(Pango::WEIGHT_BOLD);
 	list.insert(attr);
 	title_label.set_attributes(list);
-	title_label.set_alignment(Gtk::ALIGN_START,Gtk::ALIGN_CENTER);
+	title_label.set_hexpand();
+	title_label.set_halign(Gtk::ALIGN_START);
+	title_label.set_valign(Gtk::ALIGN_CENTER);
 
-	// Layer name
 	id_label.set_label(_("Name:"));
-	id_label.set_alignment(Gtk::ALIGN_START,Gtk::ALIGN_CENTER);
-	id_entry.set_hexpand();
+	id_label.set_halign(Gtk::ALIGN_START);
+	id_label.set_valign(Gtk::ALIGN_CENTER);
+	SPACING(id_gap, GAP);
```

Comment:
https://developer.gnome.org/gtkmm/3.24/classGtk_1_1Alignment.html#details

> Use Widget alignment and margin properties instead.

Can you update this part also, please?

Reply:
The `SPACING` part

Reply:
This `SPACING` approach is used a lot in the Toolboxes, maybe we should replace it everywhere as a separate issue?

Reply:
👌